Frontend/src/Components/Graph/graph-data.json

```diff
@@ -206,42 +206,42 @@
         {
           "key": "linear_accel_x",
           "name": "linear_accel_x",
-          "unit": "m/s^2",
+          "unit": "m/s²",
```

Comment:
Are you sure this will behave correctly on all platforms and not cause issues down the road?

Reply:
![image](https://user-images.githubusercontent.com/72111610/230811689-76182fb4-f6a6-4532-b526-c8bb9d649131.png)